### DISCUSSION TITLE
Uploader, Analytics and Search functionalities (Waitlist)

### DISCUSSION THREAD

**Discussion by piszczu4** [vidstack/player#1023] [General]
Is there somewhere more information regarding these extra features? When they will be ready to use and what exactly can we expect? For example, will "Uploader" handle only videos or any kind of files will be possible to upload possibly with some extra features like editing?

**Comment by mihar-22** [comment]
It's honestly too early to say. They're features we intend on building out over next year with the intention of having the CMS, storage, streaming, and Next.js integration ready in the first quarter of 2024 (somewhere between Jan-Mar). The uploader may come earlier depending on some contract work, and analytics will probably be worked on in the third quarter of 2024 if I had to guess.

The uploader will most likely support a range of media file types (not just video), but I can't say without some detailed planning. Editing sounds like a nice plugin but again no idea until we get there.
